bridge/contracts/Federation.sol

```diff
@@ -220,9 +220,9 @@ contract Federation is Initializable, UpgradableOwnable {
     function removeMember(address _oldMember) external onlyOwner
     {
         require(_oldMember != NULL_ADDRESS, "Federation: Empty member");
-        require(isMember[_oldMember], "Federation: Member doesn't exists");
-        require(members.length > 1, "Federation: Can't remove all the members");
-        require(members.length - 1 >= required, "Federation: Can't have less than required members");
+        require(isMember[_oldMember], "Federation: Member not exists");
```

Comment:
I'd change 'not exists' to 'does not exist' or 'doesn't exist'

Reply:
"Federation: Member doesn't exists" -> 33